akka-projection-core/src/main/scala/akka/projection/javadsl/SourceProvider.scala

```diff
@@ -16,4 +18,6 @@ trait SourceProvider[Offset, Envelope] {
 
   def extractOffset(envelope: Envelope): Offset
 
+  def verifyOffset(offset: Offset): OffsetVerification = Success
```

Comment:
make it an `abstract class SourceProvider` in javadsl, since this default impl will not work

Reply:
:+1: